src/Components/Aspire.Orleans.Client/AspireOrleansServerExtensions.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using Aspire.Orleans.Shared;
+using Azure.Data.Tables;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Orleans.Clustering.AzureStorage;
+using Orleans.Configuration;
+using Orleans.Runtime;
+using static Aspire.Orleans.Shared.OrleansServerSettingConstants;
+
+namespace Aspire.Orleans.Client;
+
+public static class AspireOrleansClientExtensions
+{
+    private const string OrleansConfigKeyPrefix = "Orleans";
+
+    public static IHostApplicationBuilder UseAspireOrleansClient(this IHostApplicationBuilder builder)
```

Comment:
Use vs Add?

Reply:
For client, it might make sense to use `Add` instead of `Use` (you could have multiple Orleans client)... I am fine with either of those.